README-ruRU.md

```diff
@@ -2213,15 +2212,13 @@
   Устаревший комментарий гораздо хуже отсутствующего комментария.
   <sup>[[ссылка](#comment-upkeep)]</sup>
 
-<!--- @FIXME --->
 > Хороший код подобен хорошей шутке: он не нуждается в пояснениях. <br>
 > -- Рус Ольсен (Russ Olsen)
-> &mdash; old programmers maxim, through [Russ Olsen](http://eloquentruby.com/blog/2011/03/07/good-code-and-good-jokes/)
+> &mdash; афоризм старого программиста, отсюда [Russ Olsen](http://eloquentruby.com/blog/2011/03/07/good-code-and-good-jokes/)
 
-<!--- @FIXME --->
 * <a name="refactor-dont-comment"></a>
   Не пишите комментарии для объяснения плохого кода. Перепишите код, чтобы он
-  говорил сам за себя. ("Do or do not &mdash; there is no try." Yoda)
+  говорил сам за себя.
   <sup>[[ссылка](#refactor-dont-comment)]</sup>
```

Comment:
Ok, maybe this is more polite? )
![is that better](https://cloud.githubusercontent.com/assets/3630353/19965479/392ce0e8-a1d8-11e6-8c80-b826a758087b.png)